assets/js/script.js

```diff
@@ -0,0 +1,101 @@
+const carouselList = document.querySelector(".carousel-list"),
+  carouselItems = document.querySelectorAll(".carousel-list__item"),
+  carouselWrap = document.querySelector(".carousel-wrap"),
+  btnNext = document.querySelector(".carousel-next"),
+  btnPrev = document.querySelector(".carousel-prev"),
+  pagination = document.querySelectorAll(".pagination-bullet");
+
+let carouselWrapWidth = carouselWrap.offsetWidth,
+  carouselItemsLength = carouselItems.length,
+  carouselIemsActiveIndex = 0,
+  isResizing = false;
+
+// Add or disable switch transition
+function addTransition() {
+  !isResizing
+    ? (carouselList.style.transition = "transform 0.5s ease-in-out")
+    : (carouselList.style.transition = "");
+}
+
+// Set active class to carousel item
+function addActiveToItem() {
+  carouselItems.forEach((item, index) => {
+    index === carouselIemsActiveIndex
+      ? item.classList.add("carousel-list__item--active")
+      : item.classList.remove("carousel-list__item--active");
+  });
+}
+// Add/remove class to pagination bullet
+function setActivePagination() {
+  pagination.forEach((item, index) => {
+    index === carouselIemsActiveIndex
+      ? item.classList.add("pagination-bullet--active")
+      : item.classList.remove("pagination-bullet--active");
+  });
+}
+
+// Navigate carousel on pagination click
+function moveCarouselOnPagination() {
+  pagination.forEach((item, index) => {
+    item.addEventListener("click", () => {
+      carouselIemsActiveIndex = index;
+      moveCarousel();
+    });
+  });
+}
+
+// Move carousel to selected item
+function moveCarousel() {
+  let transformValue = carouselIemsActiveIndex * carouselWrapWidth;
+  carouselList.style.transform =
+    "translate3d(-" + transformValue + "px, 0px, 0px)";
+  addActiveToItem();
+  addTransition();
+  setActivePagination();
+  setButtonAttr();
+}
+
+// Move carousel to direction
+function moveTo(direction) {
+  if (direction === "forward") {
+    carouselIemsActiveIndex++;
+    carouselIemsActiveIndex > carouselItemsLength - 1
+      ? (carouselIemsActiveIndex = carouselItemsLength - 1)
+      : "";
```

Comment:
Same here :)

Reply:
You mean do it in old fashion? :)

`if (carouselIemsActiveIndex > carouselItemsLength - 1) {
      carouselIemsActiveIndex = carouselItemsLength - 1;
    }`

Reply:
Yes, kinda.. Maybe we could lift the condition up as well ;)

```js
const isFirst = carouselItemsActiveIndex === 0;
const isLast = carouselItemsActiveIndex === carouselItemsLength - 1;

if (direction === 'forward' && !isLast) {
  carouselItemsActiveIndex += 1;
} else if (direction === 'back' && !isFirst) {
  carouselItemsActiveIndex -= 1;
}
```

Reply:
Yes, this is way better. And answers the question: What are "truthy" and "falsy" values? **Thanks.**